connectors/concur/src/main/java/com/vmware/connectors/concur/response/ConcurResponse.java

```diff
@@ -17,8 +17,11 @@
 @XmlRootElement(name = "ActionStatus", namespace = "http://www.concursolutions.com/api/expense/expensereport/2011/03")
 @XmlAccessorType(XmlAccessType.FIELD)
 @AutoProperty
+@SuppressWarnings("PMD.UnnecessaryConstructor")
 public class ConcurResponse implements Serializable {
 
+    private static final long serialVersionUID = 1L;
```

Comment:
Why does this class implement `Serializable`?

Reply:
While I was writing integration tests, I got an error mentioning that ConcurResponse should implement Serializable. I have removed it for now.. I am going to test concur connector tomorrow.. I will come to know whether ConcurConnector should implement Serializable..